api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirObservationServiceImpl.java

```diff
@@ -22,29 +22,35 @@
 import lombok.AccessLevel;
 import lombok.Setter;
 import org.hl7.fhir.r4.model.Observation;
+import org.openmrs.Obs;
 import org.openmrs.module.fhir2.api.FhirObservationService;
+import org.openmrs.module.fhir2.api.dao.FhirDao;
 import org.openmrs.module.fhir2.api.dao.FhirObservationDao;
 import org.openmrs.module.fhir2.api.translators.ObservationTranslator;
+import org.openmrs.module.fhir2.api.translators.OpenmrsFhirTranslator;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
 @Component
 @Transactional
 @Setter(AccessLevel.PACKAGE)
-public class FhirObservationServiceImpl implements FhirObservationService {
+public class FhirObservationServiceImpl BaseFhirService<Observation, org.openmrs.obs>implements FhirObservationService {
```

Comment:
You forgot to add `extend`. It should be:
```java
public class FhirObservationServiceImpl extends BaseFhirService<Observation, org.openmrs.Obs>implements FhirObservationService
```

Reply:
This is the main cause of your Travis failure

Reply:
ohh right thanks i forgot that, let me correct it